src/neptune/new/utils.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2022, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+__all__ = ["stringify_unsupported"]
+
+from typing import (
+    List,
+    Mapping,
+    Union,
+)
+
+from neptune.new.internal.utils.stringify_value import StringifyValue
+
+
+def stringify_unsupported(value) -> Union[StringifyValue, Mapping, List]:
+    if isinstance(value, dict):
+        return {k: stringify_unsupported(v) for k, v in value.items()}
+
+    if isinstance(value, list):
```

Comment:
Maybe we should use `Collection` or `Iterable`?

Reply:
I know but I'm not sure: what do I have to return in such a case? A map generator or something?